Assets/MixedRealityToolkit/_Core/Definitions/InputSystem/MixedRealityInputActionsProfile.cs

```diff
@@ -11,6 +13,34 @@ namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.InputSystem
     [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Mixed Reality Input Actions Profile", fileName = "MixedRealityInputActionsProfile", order = 1)]
     public class MixedRealityInputActionsProfile : ScriptableObject
     {
+        private readonly string[] defaultInputActions =
+{
+            "Select",
+            "Menu",
+            "Grip",
+            "Pointer",
+            "Walk",
+            "Look",
+            "Interact",
+            "Pickup",
+            "Inventory",
+            "ConversationSelect"
+        }; // Examples only, to be refined later.
+
+        private readonly AxisType[] defaultInputActionsAxis =
```

Comment:
Could we possibly rename this from AxisType to ControlType? That would allow us to have basic buttons, axes, analog triggers, etc.

Reply:
See above, the terminology is correct